Ginger/Ginger/Activities/ActivityEditPage.xaml.cs

```diff
@@ -177,6 +177,17 @@ public bool ShowAsWindow(eWindowShowStyle windowStyle = eWindowShowStyle.Dialog,
                     winButtons.Add(undoBtnAnalyzer);
                     winButtons.Add(saveBtnAnalyzer);
                     break;
+                case General.RepositoryItemPageViewMode.Standalone:
+                    title = "Edit " + GingerDicser.GetTermResValue(eTermResKey.Activity);
+                    Button saveButton = new Button();
+                    saveButton.Content = "Save";
+                    saveButton.Click += new RoutedEventHandler(saveBtn_Click);
+                    Button undoButton = new Button();
+                    undoButton.Content = "Undo & Close";
+                    undoButton.Click += new RoutedEventHandler(undoBtn_Click);
+                    winButtons.Add(undoButton);
+                    winButtons.Add(saveButton);
+                    break;
                 case General.RepositoryItemPageViewMode.View:
```

Comment:
why "Standalone" was needed to be added? for which scenario it is needed? it supposed to be Standalone only when exsit in Shared Repository and you already have sace for it.

Reply:
the same as above,
also the shared repository is not correct because of the message shown and the saving process.